rasa_core/training/generator.py

```diff
@@ -379,7 +397,7 @@ def _subsample_trackers(
         """Subsample the list of trackers to retrieve a random subset."""
 
         # if flows get very long and have a lot of forks we
-        # get into trouble by collecting to many trackers
+        # get into trouble by collecting too many trackers
```

Comment:
let's add `max_number_of_trackers` as an input parameter, to keep this method general, and pass `self.config.max_number_of_augmented_trackers` when we call it